sorc/grid_tools.fd/filter_topo.fd/filter_topo.F90

```diff
@@ -1144,6 +1147,42 @@ subroutine fill_cubic_grid_halo(data, data2, halo, ioff, joff, sign1, sign2)
 
   end subroutine fill_cubic_grid_halo
 
+  !> This routine extrapolate geolat_c and geolon_c halo points for the
+  !! regional standalone grid. Halo points are needed for dxc and dyc
+  !! calculation.
+  !!
```

Comment:
Replace the ??? with the variable definition. The ??? was a placeholder so we could get a basic doxygen capability in this code.  And use your full name and work center - "NCEP/EMC" - after 'author'.